introduction.md

```diff
@@ -1,2 +1,4 @@
 Hi. This is an introduction. I really like things. I really, really do. 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?